### DISCUSSION TITLE
SAE install with Docker or Docker-Compose

### DISCUSSION THREAD

**Discussion by MatzeG** [camueller/SmartApplianceEnabler#598] [Installation und Betrieb]
Hello 
I try to install SAE on my PI on two systems I get the error "failed to deploy a stack: sae Pulling no matching manifest for linux/arm64/v8 in the manifest list entries" I don´t know what I can do or what i´m doing wrong?

Linux raspi01 6.1.0-rpi8-rpi-v8 #1 SMP PREEMPT Debian 1:6.1.73-1+rpt1 (2024-01-25) aarch64 GNU/Linux
Docker version 20.10.24+dfsg1, build 297e128

Any ideas

Thx

**Comment by camueller** [comment]
Docker image is only availabe for 32bit arm v7, not v8.
